src/alt/index.js

```diff
@@ -77,6 +42,51 @@ class Alt {
     return makeAction(this, 'global', name, implementation, obj)
   }
 
+  createUnsavedStore(StoreModel, ...args) {
+    StoreModel.config = fn.assign({
+      invisible: true
+    }, StoreModel.config)
+    return this.createStore(StoreModel, null, ...args)
+  }
+
+  createStore(StoreModel, iden, ...args) {
+    // pull all prototype methods out
+    const proto = utils.getInternalMethods(StoreModel, true)
+
+    // force inheritance on store
+    utils.inherit(StoreModel, Store, Object.keys(proto).reduce((obj, name) => {
+      obj[name] = {
+        value: StoreModel.prototype[name],
+        enumerable: false,
+        writable: true,
+        configurable: true
+      }
+      return obj
+    }, {}))
+
+    // setup the a container store which will call Store first
+    // then your store
+    function FauxStore() {
+      Store.prototype.constructor.apply(this, arguments)
+      StoreModel.prototype.constructor.apply(this, arguments)
+    }
+    utils.inherit(FauxStore, StoreModel)
+
+    if (iden) FauxStore.displayName = iden
+    return this.register(...args)(FauxStore)
+  }
+
+  register(...args) {
```

Comment:
Is the intended syntax something like `alt.register(ctorArg1, ctorArg2)(Store)`? I feel like `alt.register(Store, ctorArg1, ctorArg2)` looks a bit better. Although that wouldn't work as a decorator... but I can't think of a good reason to write code that looks like

```js
@alt.register(arg1, arg2)
class MyStore extends Store { ... }
```

With the latter above I think you just naturally get

```js
@alt.register
class MyStore extends Store { ... }
```

which ought to be good enough for any sane use case.